web/handlers.go

```diff
@@ -231,7 +231,7 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		// Set content security policy. This is also specified in the root.html of the webapp in a meta tag.
 		w.Header().Set("Content-Security-Policy", fmt.Sprintf(
 			"frame-ancestors 'self'; script-src 'self' *.riffplatform.com *.riffedu.com *.riffai.cloud cdn.rudderlabs.com%s%s%s%s",
-			" *.googletagmanager.com 'sha256-K4FCg61IQARpz2f8JyoJsmQfYJOCjZGl0TkKzyDxppE='",
+			" https://cdn.userway.org/ 'sha256-SW4mlJLRxlBpckfBUvfFo6QJIMVl74UJsWvLxoUiW28=' *.googletagmanager.com 'sha256-K4FCg61IQARpz2f8JyoJsmQfYJOCjZGl0TkKzyDxppE='",
```

Comment:
I'd suggest this, clearly separating out the new userway content rather than prepending it to the google analyics:
```
		// Set content security policy. This is also specified in the root.html of the webapp in a meta tag.
		w.Header().Set("Content-Security-Policy", fmt.Sprintf(
			"frame-ancestors 'self'; script-src 'self' *.riffplatform.com *.riffedu.com *.riffai.cloud cdn.rudderlabs.com%s%s%s%s%s",
			" *.googletagmanager.com 'sha256-K4FCg61IQARpz2f8JyoJsmQfYJOCjZGl0TkKzyDxppE='",
			" https://cdn.userway.org/ 'sha256-SW4mlJLRxlBpckfBUvfFo6QJIMVl74UJsWvLxoUiW28='",
			cloudCSP,
			h.cspShaDirective,
			devCSP,
		))
```

Reply:
Good idea!  I did this and pushed that up.